lib/PoolOptions.js

```diff
@@ -4,40 +4,45 @@ const PoolDefaults = require("./PoolDefaults");
 
 class PoolOptions {
   /**
-   * @param {Object} config
+   * @param {Object} opts
```

Comment:
From here I fixed name mismatches, missing parameters and defaults and updated doc reference.